src/index.js

```diff
@@ -129,12 +126,14 @@ exports.createFromB58String = function (str) {
   return new Id(new Buffer(base58.decode(str)))
 }
 
+// Public Key input will be a buffer
```

Comment:
@nginnever can you add a test so that:

```javascript
const id1 = peerId.create()
const id2 = peerId.createFromPublicKey(id1.pubkey)
expect(id2.id).to.equal(id1.id)
```